src/client/components/localisation/view_model.ts

```diff
@@ -52,6 +55,13 @@ export class LocalisationViewModel {
   }
 
   @computed
+  private get balls(): Object3D[] {
+    return this.model.robots
+      .filter(robotModel => robotModel.visible)
```

Comment:
yeah I had a similar thought, but instead I was going to make `get robot(): Object3D[]` to return `LocalisationRobotModel[]` and do this:

```TS
this.robots.forEach(ball => {
  scene.add(robot.robot)
  scene.add(robot.ball)
})
```